store/subscription.ts

```diff
@@ -0,0 +1,39 @@
+import Vue from 'vue'
```

Comment:
@roiLeo @vikiival I have add subscription in Vuex store, so this will handle all of the subscriptions. `singleSubscribeMixin` will not be useful in our case since we need to subscribe in `userBalance.ts` plugin and I think the mentioned mixin will not be helpful in that case. I think this is the best way to handle subscriptions.

Reply:
And what about unsubscribe?